pages/en/network-transition-faq.mdx

```diff
@@ -70,27 +70,30 @@ The Graph's decentralized network is 60-90% less expensive than running dedicate
+### When will the decentralized network support my preferred network?
 
-There is no set timeline per chain, they will be dictated by Indexer readiness via the [MIPs program](https://thegraph.com/migration-incentive-program/) where new chains are tested by Indexers. As new chains are supported on the network, users will receive ample notification to prepare for migration. Core devs and contributors to The Graph ecosystem are working to implement support for more chains as soon as possible.
+There is no set timeline per network, they will be dictated by Indexer readiness via the [MIPs program](https://thegraph.com/migration-incentive-program/) where new networks are tested by Indexers. As new networks are supported on the network, users will receive ample notification to prepare for migration. Core devs and contributors to The Graph ecosystem are working to implement support for more networks as soon as possible.
 
 ### Is Ethereum mainnet entering Phase 1 of the network transition process?
 
 While Ethereum was initially anticipated to begin transition off of the hosted service by the end of Q3 2022, this has been [postponed](https://thegraph.com/blog/transitioning-to-decentralized-graph-network) to address user feedback. Additional improvements to user experience, billing, and other fulfillments of user requests will drive Ethereum's hosted service transition timeline. Stay up to date on when Ethereum will enter The Sunray phase via the integration status tracker below and via [The Graph Twitter.](http://www.twitter.com/graphprotocol)
 
 ### The Graph Network integration status tracker
 
-The table below illustrates where each chain is in the network integration process. If your preferred chain is not yet listed, integration has not yet begun, and that chain is still fully supported by The Graph's hosted service.
+The table below illustrates where each network is in the network integration process. If your preferred network is not yet listed, integration has not yet begun, and that network is still fully supported by The Graph's hosted service.
 
-> This table will not include test chains, which remain free in [Subgraph Studio](https://thegraph.com/studio/).
+> This table will not include test networks, which remain free in [Subgraph Studio](https://thegraph.com/studio/).
 
-| Chain / Network | Announcing integration on The Graph Network | Network Integration complete | Phase 1: disable new subgraphs on hosted service | Phase 2: disable subgraph upgrades on hosted service | Phase 3: disable subgraphs on hosted service |
+| Network | Announcing integration on The Graph Network | Network Integration complete | Phase 1: disable new subgraphs on hosted service | Phase 2: disable subgraph upgrades on hosted service | Phase 3: disable subgraphs on hosted service |
 | --- | :-: | :-: | :-: | :-: | :-: |
 | Ethereum | ✓ | ✓ |  |  |  |
 | Gnosis (formerly xDAI) | ✓ | ✓\* |  |  |  |
-| Polygon | ✓ |  |  |  |  |
+| Polygon | ✓ | ✓\* |  |  |  |
```

Comment:
Polygon have been announced, but not yet been integrated. (Second checkmark should be removed) We can also add Optimism and Fantom as Announced, but not yet integrated.

| Polygon | ✓ | | | | |
| Optimism | ✓ | | | | |
| Fantom | ✓ | | | | |

Reply:
Thank you for catching that!